src/constants.ts

```diff
@@ -0,0 +1,16 @@
+import { Sort } from "./types";
+
+export const defaultLimit = 100;
+export const defaultSkip = 0;
+export const defaultProjection = ["domainid", "owner", "name"];
```

Comment:
since we've axed projection, you can remove this.

Reply:
Itll be gone once I merge